src/saving/SaveManagerGUI.cs

```diff
@@ -166,17 +172,39 @@ private void DeleteSelectedButtonPressed()
         deleteSelectedConfirmDialog.PopupCenteredMinsize();
     }
 
+    private void DeleteOldButtonPressed()
+    {
+        deleteOldConfirmDialog.DialogText =
+            "Deleting all old Auto and Quick saves cannot be undone, " +
+            "are you sure you want to permanently delete the following?\n" +
+            $" >>> {Mathf.Clamp(SaveHelper.CountSaves("auto_save").count - 1, 0, Settings.Instance.MaxAutoSaves)} " +
+            "Auto save(s)\n" +
+            $" >>> {Mathf.Clamp(SaveHelper.CountSaves("quick_save").count - 1, 0, Settings.Instance.MaxQuickSaves)} " +
+            "Quick save(s)";
+        deleteOldConfirmDialog.PopupCenteredMinsize();
+    }
+
     private void OnConfirmDeleteSelected()
     {
         GD.Print("Deleting save(s): ", string.Join(", ", Selected.Select(item => item.SaveName).ToList()));
 
         Selected.ForEach(item => SaveHelper.DeleteSave(item.SaveName));
-        deleteSelectedButton.Disabled = true;
```

Comment:
Why was this line removed?

Reply:
It was removed because since UpdateButtonsStatus was moved to _Process, after every RefreshList call the buttons are automatically updated and there is no need to disable it before the RefreshList call.

Reply:
But that happens asynchronously, so if it takes a long time to refresh the list, it would be good to disable the button until the list is refreshed.

Reply:
Ah I see, got it.